manager/api/requests.go

```diff
@@ -110,16 +110,16 @@ func (req viewResourceReq) validate() error {
 
 type listResourcesReq struct {
 	key    string
-	size   int
 	offset int
+	limit  int
 }
 
 func (req listResourcesReq) validate() error {
 	if req.key == "" {
 		return manager.ErrUnauthorizedAccess
 	}
 
-	if req.size > 0 && req.offset >= 0 {
+	if req.offset > 0 && req.limit >= 0 {
```

Comment:
While i agree with renaming `size` to `limit`, the condition was correct. You can start from the beginning (`offset >= 0`), and you must select at least 1 element (`limit > 0`).